packages/grid/test/drag-and-drop.common.js

```diff
@@ -254,6 +254,42 @@ describe('drag and drop', () => {
         });
       });
 
+      it('should add drag-source- part to all dragged rows', async () => {
+        grid.selectItem(grid.items[0]);
+        grid.selectItem(grid.items[1]);
+        fireDragStart();
+        await nextFrame();
+        for (const row of [...grid.$.items.children]) {
+          for (const cell of [...row.children]) {
```

Comment:
nit: Prefer using existing helpers over `children` API in tests since they are built to take into account exceptions like rows hidden by virtualizer and details cell.

To align with the other tests in this file, you could use the following helpers
```js
for (const row of getRows(grid.$.items)) {
  for (const cell of getRowBodyCells(row)) {
```